frontend/public/components/_overview.scss

```diff
@@ -24,3 +24,51 @@
   margin-bottom:  ($grid-gutter-width / 4);
   padding-left: 0;
 }
+
+.overview {
+  width: 100%;
+  display: flex;
+
+  .overview__body {
```

Comment:
We should generally avoid nesting CSS classes since we use BEM naming (unless we need to in specific cases for greater specificity).

Reply:
Should we leave this for @sg00dwin's follow up style fixes?

Reply:
Works for me